test/integration/puppetlabs/services/master/tasks_int_test.clj

```diff
@@ -0,0 +1,101 @@
+(ns puppetlabs.services.master.tasks-int-test
+  (:require [clojure.test :refer :all]
+            [clojure.string :as str]
+            [puppetlabs.http.client.sync :as http-client]
+            [puppetlabs.kitchensink.core :as ks]
+            [puppetlabs.puppetserver.bootstrap-testutils :as bootstrap]
+            [puppetlabs.puppetserver.testutils :as testutils]
+            [puppetlabs.trapperkeeper.testutils.bootstrap :as tk-bootstrap-testutils]
+            [puppetlabs.trapperkeeper.testutils.webserver :as jetty9]
+            [puppetlabs.services.master.master-core :as master-core]
+            [puppetlabs.services.protocols.jruby-puppet :as jruby-protocol]
+            [cheshire.core :as cheshire]
+            [me.raynes.fs :as fs]
+            [clojure.tools.logging :as log]
+            [puppetlabs.services.jruby.jruby-puppet-testutils :as jruby-puppet-testutils])
+  (:import (com.puppetlabs.puppetserver JRubyPuppetResponse JRubyPuppet)
+           (java.util HashMap)))
+
+(def test-resources-dir
+  "./dev-resources/puppetlabs/services/master/environment_classes_int_test")
```

Comment:
It's been the pattern to copy the needed resources to a directory of the name of the current test.